python/example/single_cell_extracellular_potentials.py

```diff
@@ -122,100 +117,164 @@ def probes(self, gid):
 domains = arbor.partition_load_balance(recipe, context)
 sim = arbor.simulation(recipe, domains, context)
 
-# set up sampling on probes
-schedule = arbor.regular_schedule(0.1)
+# set up sampling on probes with sampling every 1 ms
+schedule = arbor.regular_schedule(1.)
 v_handle = sim.sample(recipe.vprobe_id, schedule, arbor.sampling_policy.exact)
 i_handle = sim.sample(recipe.iprobe_id, schedule, arbor.sampling_policy.exact)
 c_handle = sim.sample(recipe.cprobe_id, schedule, arbor.sampling_policy.exact)
 
 # run simulation for 500 ms of simulated activity and collect results.
 sim.run(tfinal=500)
 
-# extract time, V_m and I_m for each compartment
+# extract time, V_m, I_m and I_c for each CV
 V_m_samples, V_m_meta = sim.samples(v_handle)[0]
 I_m_samples, I_m_meta = sim.samples(i_handle)[0]
 I_c_samples, I_c_meta = sim.samples(c_handle)[0]
 
-
 # drop recorded V_m values and corresponding meta data of
 # zero-sized CVs (branch-point potentials)
 inds = np.array([m.dist != m.prox for m in V_m_meta])
 V_m_samples = V_m_samples[:, np.r_[True, inds]]
```

Comment:
This line I do not understand, and the description on lines 134-135 only cover line 136, IIUC. Can you give a bit more detail?

Reply:
I agree this was confusing (also momentarily confused me as the the voltage probe samples at branch points while the current probe does not). The `V_m' values are only used for the plot. I'll detail this some more.